c3s_magic_wps/processes/wps_drought_indicator.py

```diff
@@ -32,6 +33,8 @@ def __init__(self):
                          default='ERA-Interim',
                          min_occurs=1,
                          max_occurs=1),
+            *reference_year_ranges((1980, 1989)),
```

Comment:
This needs the recipe template to also know about reference years, and code to pass it to the template

Reply:
https://github.com/c3s-magic/c3s-magic-wps/pull/98/commits/c4c6190a878e233229b3be9b8a2a99c260abaf31